src/collections/_documentation/guides/enrich-data/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Put your Data to Work
+sidebar_order: 4
+---
+
+The Sentry SDK captures errors, exceptions, crashes, and generally anything that goes wrong in your application in real-time.
+Once captured, the SDK will enrich the error data with contextual information that it grabs from the application's runtime and send all this data as an **event** to your Sentry account.
+
+While Sentry adds a lot of this contextual data by default, we highly encourage you to **add your custom data** through the SDK. Multiple Sentry workflows and tools can be configured and fine-tuned based on this data - so adapting those to the way you work, will go a long way in getting even more value out of Sentry.
+
+## Types of Data
+
+The SDK allows you to set various types of data that will then be attached to every event that occurs in your application. Generally, those data types are :
+
+1. **Unstructured Data** - include additional data like `Breadcrumbs`, `Extra data`, and `Custom Contexts`. While these felids are **unsearchable**, they enrich your events with information that will help you debug and resolve the associated issues.
```

Comment:
In the SDK doc, we are calling these two types of data "Predefined data" and "Custom data" - 

Data sent with every event is either predefined or custom:
- *Predefined data* is data that Sentry recognizes. This data enhances your ability to understand and investigate the source and impact of the data when viewing it in the Sentry web UI. For example, in Sentry, the concept of user helps us displays the number of unique users affected by an event.
- *Custom data* is arbitrary structured or unstructured extra data you can attach to your event

Can you adopt these terms for consistency?